microprofile/WFLY-13023_upgrade_microprofile-config_1.4.adoc

```diff
@@ -0,0 +1,58 @@
+= Upgrade MicroProfile Config to 1.4
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,config,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
```

Comment:
Can you please summarize main differences between `1.3` and `1.4` MP Config versions in this Overview section?

Reply:
I'll add a link to the release notes

Reply:
done